packages/transaction-controller/src/utils.ts

```diff
@@ -60,61 +63,11 @@ export function validateTxParams(
   txParams: TransactionParams,
```

Comment:
Apologies for the confusion, but as they are all related, I was suggesting we move this method to `validation.ts` also and have it be the only exported function from the new file as all the others are only used by it.

The long term goal is to minimise `utils.ts` until it's not needed at all as I feel it's too general for something as big as the `TransactionController`.

Reply:
ow no, so sorry that was on my end. 
I just moved `validateTxParams` and sorted the unit test as well.